controllers/storagecluster/cephcluster.go

```diff
@@ -116,9 +117,18 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return err
 	}
 
+	platform, err := r.platform.GetPlatform(r.Client)
+	if err != nil {
+		r.Log.Info("Failed to get platform")
```

Comment:
should be logged as error.

Reply:
shouldn't we return in case of error ?

Reply:
We are changing the mon failover timeout for IBM platform only, but if it fails I don't think it should be a problem.
So maybe instead of just logging it, Throwing a warning would be better.

Reply:
ok. so if we don't want to return error here, then warning should be fine and next if block should be skipped.

Reply:
Logging the error